protocol/avdl/keybase1/user.avdl

```diff
@@ -179,15 +179,20 @@ protocol user {
   */
   NextMerkleRootRes findNextMerkleRootAfterReset(UID uid, Seqno resetSeqno, ResetMerkleRoot prev);
 
-  boolean loadHasRandomPw(int sessionID, boolean forceRepoll, boolean noShortTimeout);
+  enum PassphraseState {
+    KNOWN_0,
+    RANDOM_1
+  }
```

Comment:
let's leave a note here that these end up in config.json so we can't change this without breaking users